### PR TITLE
Sabnzbd config flow documentation

### PR DESCRIPTION
**Pull request by shaiu**
## Proposed change

1. add config flow to Sabnzbd documentation
2. add required API key to Services section
2. remove old configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68138 https://github.com/home-assistant/core/pull/70981

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by shaiu** [comment]
@MartinHjelmare @frenck FYI documentation update

File: source/_integrations/sabnzbd.markdown [hunk 1]
```diff
@@ -14,107 +17,56 @@ ha_integration_type: integration
...
+
+- `pause`: Pause the download queue.
+- `resume`: Resume the download queue.
+- `set_speed`: Set the download queue speed limit.
+
+### Service `sabnzbd.pause`
+
+| Service data attribute | Optional | Description                                                                                                                                                                                 |
+|------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Sabnzbd API key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```
Comment (MartinHjelmare) [inline source/_integrations/sabnzbd.markdown]:
```suggestion
| `api_key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```

File: source/_integrations/sabnzbd.markdown [hunk 2]
```diff
@@ -14,107 +17,56 @@ ha_integration_type: integration
...
+
+### Service `sabnzbd.resume`
 
-Available services: `pause`, `resume`, `set_speed`.
+| Service data attribute | Optional | Description                                                                                                                                                                                 |
+|------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Sabnzbd API key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```
Comment (MartinHjelmare) [inline source/_integrations/sabnzbd.markdown]:
```suggestion
| `api_key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```

File: source/_integrations/sabnzbd.markdown [hunk 3]
```diff
@@ -14,107 +17,56 @@ ha_integration_type: integration
...
-| `speed`                |      yes | Sets the download speed limit. If specified as a number with no units, will be interpreted as a percent. If units are provided (e.g., 500K) will be interpreted absolutely. Defaults to 100 |
+| Service data attribute | Optional | Description                                                                                                                                                                                 |
+|------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `speed`                | yes      | Sets the download speed limit. If specified as a number with no units, will be interpreted as a percent. If units are provided (e.g., 500K) will be interpreted absolutely. Defaults to 100 |
+| `Sabnzbd API key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```
Comment (MartinHjelmare) [inline source/_integrations/sabnzbd.markdown]:
```suggestion
| `api_key`      | no       | The SABnzbd API key of the service instance (unique per SABnzbd integration).                                                                                                               |
```

**Review by MartinHjelmare — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @shaiu 👍

**Pull request merged**